pages/HotMealDayCard.js

```diff
@@ -0,0 +1,46 @@
+import { useRouter } from "next/router";
+
+function HotMealDayCard() {
+  const router = useRouter();
+
+  const handleSubmit = async (event) => {
+    event.preventDefault();
+
+    const response = await fetch("/api/create-stripe-session", {
+      body: JSON.stringify({
+        amount: 250 * 100,
+        cause: "Hot Meal for 60 Hungry People.",
+      }),
+      headers: {
+        "Content-Type": "application/json",
+      },
+      method: "POST",
+    });
+    const result = await response.json();
+    router.push(result.url);
+  };
+  return (
+    <div>
```

Comment:
I noticed you followed my pattern from HygieneKitsCard.js with the extra outer `<div>`. I'm curious to hear your thoughts on it in #19 separately. I'll accept this and we can iterate on that part later if anyone thinks it's a good idea.